app/allocation/controllers/view.js

```diff
@@ -0,0 +1,38 @@
+const presenters = require('../../../common/presenters')
+
+module.exports = function view(req, res) {
+  const {
+    allocation: {
+      prisoner_category: prisonerCategory,
+      complex_cases: complexCases,
+      date,
+      from_location: fromLocation,
+      moves_count: movesCount,
+      other_criteria: otherCriteria,
+      sentence_length: sentenceLength,
+      to_location: toLocation,
+      moves,
+    },
+  } = res.locals
+  const locals = {
+    dashboardUrl: '/allocations',
+    allocationDetails: presenters.allocationToMetaListComponent({
```

Comment:
In this instance you can just pass the allocation in, save you having to destructure it and re-build it in the argument.

The nice thing about destructing in the presenter is it will then ignore any properties it doesn't need.

It will allow you to reduce the size of this controller a lot too.